web/angular-wallet/src/app/navigation/navigation.ts

```diff
@@ -26,89 +26,89 @@ export const navigation: FuseNavigation[] = [
-                url      : 'transactions/escrow'
-            },
-            {
-                id       : 'subscriptions',
-                title    : 'Subscriptions',
-                type     : 'item',
-                icon     : 'subscriptions',
-                url      : 'transactions/subscriptions'
-            },
-            {
-                id       : 'smart_contracts',
-                title    : 'Smart Contracts',
-                type     : 'item',
-                icon     : 'computer',
-                url      : 'transactions/smart-contracts'
-            }
-        ]
-    },
-    {
-        id       : 'asset_exchange',
-        title    : 'Exchange',
-        type     : 'group',
-        children : [
-            {
-                id       : 'asset_exchange',
-                title    : 'Exchange',
-                type     : 'item',
-                icon     : 'account_balance',
-                url      : '/dex',
-                badge    : {
-                    title    : '25',
-                    translate: '25',
-                    bg       : '#F44336',
-                    fg       : '#FFFFFF'
-                }
-            },
-            {
-                id       : 'transfer_history',
-                title    : 'Transfer History',
-                type     : 'item',
-                icon     : 'history',
-                url      : '/dex/transfer-history'
-            },
-            {
-                id       : 'my_assets',
-                title    : 'My Assets',
-                type     : 'item',
-                icon     : 'pie_chart',
-                url      : '/dex/my-assets'
-            },
-            {
-                id       : 'open_orders',
-                title    : 'Open Orders',
-                type     : 'item',
-                icon     : 'list_alt',
-                url      : '/dex/open-orders'
-            },
-            {
-                id       : 'issue_asset',
-                title    : 'Issue Asset',
-                type     : 'item',
-                icon     : 'add_box',
-                url      : '/dex/issue-asset'
-            }
+            // {
+            //     id       : 'escrow',
+            //     title    : 'Escrow',
+            //     type     : 'item',
+            //     icon     : 'check_circle',
+            //     url      : 'transactions/escrow'
+            // },
+            // {
+            //     id       : 'subscriptions',
+            //     title    : 'Subscriptions',
+            //     type     : 'item',
+            //     icon     : 'subscriptions',
+            //     url      : 'transactions/subscriptions'
+            // },
+            // {
+            //     id       : 'smart_contracts',
+            //     title    : 'Smart Contracts',
+            //     type     : 'item',
+            //     icon     : 'computer',
+            //     url      : 'transactions/smart-contracts'
+            // }
         ]
     },
+    // {
```

Comment:
all commented?

Reply:
Yeah for now - we can uncomment as we build these features. I dont see us having them all done for MVP.